src/index.css

```diff
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.todo__wrapper {
+  width: 50%;
+  height: 70vh;
+  max-height: 70vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: center;
+  border: 0.5px solid #ccc;
+  box-shadow: 0 0 5px 0 #242222bf;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 1rem;
+}
+
+header .refresh-icon {
+  cursor: pointer;
+}
+
+header p {
+  font-weight: 500;
+  font-style: italic;
+}
+
+form {
+  width: 100%;
+}
+
+form input {
+  width: 100%;
+  padding: 0.5rem 1rem;
+}
+
+.todoList__wrapper {
+  width: 100%;
+}
+
+.todo {
+  display: flex;
+  justify-content: space-between;
+  column-gap: 1rem;
+  align-items: center;
+  width: 100%;
+  border-bottom: 0.5px solid #ccc;
+  padding: 0.5rem 1rem;
+}
+
+.todo p {
+  flex: 1;
+}
+
+.todo .description {
+  justify-self: flex-start;
+}
+
+.dots {
+  cursor: pointer;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.dots:hover div {
+  background-color: #212121;
+}
+
+.dot {
+  height: 3px;
+  width: 3px;
+  background-color: #ccc;
+  border-radius: 50%;
+}
+
+.clear__btn {
+  padding: 0.5rem 1rem;
+  font-size: 1rem;
+  margin-top: 0.5rem;
+}
```

Comment:
- [ ] You did a good job on the styles but you need to work on the `Clear all completed` button. Please make sure the button has the same width as the tasks container and matches the given design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/213185710-c37d3095-0798-4cb8-827d-c91302a22096.png)

### Given design

![image](https://user-images.githubusercontent.com/83803414/213185657-54f430f4-d2cf-456b-a715-318d68d06275.png)